README.md

```diff
@@ -44,15 +44,29 @@
 
 To name functions, you can use some keywords that everyone will understand, like those ones :
 
-`updateSomething` : should perform server-side updates
+`fetchSomething` : should access data by making a server side call
 
 `getSomething` : should access data without making a server side call
 
-`fetchSomething` : should access data by making a server side call
+`findSomething` : like the get, but with arguments
 
 `normalizeSomething` : should transform data to conform to a given schema or to make sure that is coherent with other documents
 
-`harmonizeSomething` : should tweak data a bit to work well with other items
+`saveSomething` : creates or updates a document
+
+`updateSomething` : should perform server-side updates
+
+`hasSomething` or `isSomething` : should describe a boolean variable
+
+`computeSomething` : create something from other elements
+
+`makeSomething` : create something from scratch
+
+`doSomethingAndForget` : to convert an async func `doSomething` to a sync one `doSomethingAndForget`
+
+`format` : from an object, returns a string
```

Comment:
why not... toString()? Because `format` is too much generic to only return a string

Reply:
on s'en sert par exemple parfois pour formater des dates. A mon avis sauf pour les isToto ou hasToto qui retourne la plupart du temps des bool, on peut éviter dans cette doc de "forcer" un type concernant le retour

Reply:
we could simply use the `computeSomething` instead of `formatSomething` if we do not force the return type to string

Reply:
we decide to remove `format` but if you use it often in the future don't hesitate to open a PR